src/MoBi.Engine/Sbml/UnitDefinitionImporter.cs

```diff
@@ -48,37 +53,39 @@ public class UnitDefinitionImporter : SBMLImporter, IStartable, IUnitDefinitionI
    {
       private readonly IMoBiDimensionFactory _moBiDimensionFactory;
       private readonly IDictionary<int, Unit> _baseUnitsDictionary;
-      private readonly IDictionary<string, UnitConvertionInfo> _unitConvertionDictionary;
-      private IDimensionFactory _dimensionFactory;
```

Comment:
why this guy here sinec we have a MoBiDimensionFactory (which is a specification of IDimensionFActory)